deps.edn

```diff
@@ -1,5 +1,4 @@
 {:paths ["src"]
- :deps {org.babashka/sci {:mvn/version "0.2.8"}
+ :deps {org.babashka/sci #_{:local/root "../babashka/sci"} {:mvn/version "0.2.8"}
```

Comment:
Intentional: this makes it easier for me to locally repro, I'd like to leave this in.